index.js

```diff
@@ -205,3 +219,40 @@ class FindDummyAppFiles extends Plugin {
     fs.writeFileSync(path.join(this.outputPath, 'app-files.js'), `export default ${pathsString};`);
   }
 }
+
+class AutoExportAddonToApp extends Plugin {
+  build() {
+    let addonPath = this.inputPaths[0];
+
+    // Components
+    walkSync(path.join(addonPath, 'components'), { directories: false })
+      .forEach(addonFile => {
+        let module = addonFile.replace('/component.js', '');
+        let file = path.join(this.outputPath, 'components', `${module}.js`);
+        ensureDirectoryExistence(file);
+        fs.writeFileSync(file, `export { default } from 'ember-cli-addon-docs/components/${module}/component';`);
+      });
+
+    // Non-pods modules (slightly different logic)
+    [ 'adapters', 'controllers', 'models', 'services', 'transitions' ].forEach(moduleType => {
+      let addonFiles = walkSync(path.join(addonPath, moduleType), { directories: false });
+
+      addonFiles.forEach(addonFile => {
+        let module = addonFile.replace('.js', '');
+        let file = path.join(this.outputPath, moduleType, `${module}.js`);
+        ensureDirectoryExistence(file);
+        fs.writeFileSync(file, `export { default } from 'ember-cli-addon-docs/${moduleType}/${module}';`);
+      });
+    });
+
+  }
```

Comment:
Will this handle `routes`? I saw you removed that from the `app` tree, but I'm not sure I see how that reexport gets generated here.

Reply:
Woops! Good catch... some acceptance tests woulda caught that